src/Employer/Employer.Web/Controllers/Part1/TitleController.cs

```diff
@@ -21,27 +24,55 @@ public TitleController(TitleOrchestrator orchestrator)
         }
         
         [HttpGet("create-vacancy", Name = RouteNames.CreateVacancy_Get)]
-        public IActionResult Title()
+        public async Task<IActionResult> Title([FromRoute] string employerAccountId)
         {
-            var vm = _orchestrator.GetTitleViewModel();
+            var vm = await _orchestrator.GetTitleViewModel(employerAccountId);
             vm.PageInfo.SetWizard();
+            GetReferredDataFromTempData(vm);
+            vm.ReturnToMALinkText = SetBackText(vm);
```

Comment:
This is odd. vm.ReturnToMALinkText = SetBackText({required variables})

Also, as discussed, the view doesn't care about MA. rename to `BackLinkText`